src/Entities/Minecart.h

```diff
@@ -78,6 +78,8 @@ class cMinecart :
 	void SnapToRail(NIBBLETYPE a_RailMeta);
 	/** Tests if a solid block is in front of a cart, and stops the cart (and returns true) if so; returns false if no obstruction */
 	bool TestBlockCollision(NIBBLETYPE a_RailMeta);
+	/** Tests if a solid block is at a specific offset of the minecart position */
+	bool IsBlockAtOffset(int a_XOffset, int a_YOffset, int a_ZOffset);
```

Comment:
Please rename to `IsSolidBlockAtOffset` to make it clear what the function does.